examples/tensorflow/README.md

```diff
@@ -61,11 +61,17 @@ This is Tensorflow implementation of Neural Turing Machine. This implementation
 Example of a deep convolutional generative adversarial network. A network able to reproduce detailed images.
 Train data can be supplied in a folder with name input or passed as arg with --input_dir, Tweak the values if necessary. Or it will use MNIST dataset as default.
 
+
 8. **yolo-tensorflow**
 
 YOLO (You Only Look Once) is a state-of-the-are, real-time object detection system. This example trained a yolo model with pascal-voc 2007 data.
```

Comment:
should that be State-of-the-Art ?

Reply:
I made a pull request of 312 lines where ONE of these 312 is clearly a typographical error and, since I didn't disable "Allow edits from maintainers.", is easily fixed by anyone who reviews this pull request. If your only objective here is making a joke, I should say you are at the wrong place.